ci-operator/config/openstack-k8s-operators/openstack-operator/openstack-k8s-operators-openstack-operator-master.yaml

```diff
@@ -33,6 +33,31 @@ images:
     RUN chmod +x /usr/local/bin/oc
   from: src
   to: openstack-operator-ci-build-deploy
+- dockerfile_literal: |
```

Comment:
@arxcruz once we merge this patch, we can work on moving this Dockerfile to openstack-k8s-operators-ci. So we can just pull the images, instead of always building it on every job. But first we can merge at it is.